### PR TITLE
repl: reproduce the void print function error

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
Fixes https://github.com/vlang/v/issues/13371

Signed-off-by: Vincenzo Palazzo <vincenzopalazzodev@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -105,3 +105,5 @@ thirdparty/stdatomic/nix/cpp/*.h
 
 # ignore VLS log
 vls.log
+
+hi
```
Comment (spytheman) [inline .gitignore:109]: hello :-)

Comment (vincenzopalazzo) [reply .gitignore]: Sorry to many command line together :)

**Review by spytheman — APPROVED** [review]
Excellent work.

**Comment by vincenzopalazzo** [comment]
Thanks for your improvement, the code now is much better

**Pull request merged**